File: lesson_5_1/app/src/main/java/com/kalashnyk/denys/simplemvpapp/data/pojo/response/UserResponse.java
package com.kalashnyk.denys.simplemvpapp.data.pojo.response;

import com.kalashnyk.denys.simplemvpapp.data.database.entity.UserEntity;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class UserResponse {
    @SerializedName("records")
    private List<UserEntity> mRecords;

    public List<UserEntity> getRecords() {
        return mRecords;
    }

    public void setRecords(List<UserEntity> mRecords) {
        this.mRecords = mRecords;
    }
}

File: lesson_7/app/src/main/java/com/kalashnyk/denys/kotlinsample/repository/database/pojo/UserResponse.kt
package com.kalashnyk.denys.kotlinsample.repository.database.pojo

import com.google.gson.annotations.SerializedName
import com.kalashnyk.denys.kotlinsample.repository.database.entity.UserEntity

class UserResponse {
    @SerializedName("records")
    var records: List<UserEntity>? = null
}